getMultiStarts.m

```diff
@@ -117,61 +117,68 @@
         fprintf(' \nOptimization:\n=============\n');
     case 'silent' % no output
         % Force fmincon to be silent.
-        if strcmp(options.localOptimizer, 'fmincon')
+        if (strcmp(options.localOptimizer, 'fmincon') || strcmp(options.localOptimizer, 'lsqnonlin'))
```

Comment:
can we simply check `isfield(options.localOptimizerOptions,'Display')`? Should be more robust for further extensions